v2src/CMakeLists.txt

```diff
@@ -168,13 +168,14 @@ target_include_directories(aotriton_v2 PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../inclu
 target_include_directories(aotriton_v2 PUBLIC ${CMAKE_BINARY_DIR}/include)  # for <build>/include/aotriton/config.h. Code should use <aotriton/config.h>
 target_include_directories(aotriton_v2 PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 target_include_directories(aotriton_v2 PRIVATE ${CMAKE_CURRENT_LIST_DIR}/../third_party/incbin)
-target_compile_options(aotriton_v2 PRIVATE -fPIC --no-offload-arch=all)
+target_compile_options(aotriton_v2 PRIVATE -fPIC)
 if(AOTRITON_BUILD_FOR_TUNING)
   target_compile_definitions(aotriton_v2 PRIVATE -DAOTRITON_BUILD_FOR_TUNING=1)
 else(AOTRITON_BUILD_FOR_TUNING)
   target_compile_definitions(aotriton_v2 PRIVATE -DAOTRITON_BUILD_FOR_TUNING=0)
 endif(AOTRITON_BUILD_FOR_TUNING)
-target_link_libraries(aotriton_v2 lzma_interface)
+target_link_libraries(aotriton_v2 PRIVATE lzma_interface)
+target_link_libraries(aotriton_v2 PUBLIC hip::host hip::amdhip64)
```

Comment:
We need to introduce an explicit dependency on the HIP targets because we are not using the hipcc compiler as the default compiler anymore: https://github.com/ROCm/aotriton/pull/51/files#diff-3c713483b9080c3c648c0ac4fb576cc9fbe0ad2b2b4f661637939993fbbc6fbcR19